enhancements/api-review/custom-tags-aws.md

```diff
@@ -101,7 +88,73 @@ The userTags field is intended to be set at install time and is considered immut
 
 If the userTags field is changed post-install, there is no guarantee about how an in-cluster operator will respond to the change. Some operators may reconcile the change and change tags on the AWS resource. Some operators may ignore the change. However, if tags are removed from userTags, the tag will not be removed from the AWS resource.
 
-#### Existing drawbacks
+### Update design details
+
+The `experimentalPropagateUserTags` field is changed to `propagateUserTags` for GA version.
```

Comment:
not clear if it's implied but we should have a few releases where we keep supporting the old field name too, before we drop it entirely, so consumers have a change to change their scripts/templates/configs.

this should also be covered in the migration section.

Technically we are under no obligation to provide such a transition since it was an experimental field, but it's basically no cost to us and avoids operations teams from being surprised if we stop respecting the experimental flag and they don't get the tag propagation behavior they expected, suddenly.

Reply:
For the new field `propagateUserTags` , Can the flag used to propagate user flags `propagateUserTags` be removed and made a default propagation of user-defined tags ? Is there a scenario where a switch like `propagateUserTags` is useful ?